src/tests/test_example_block.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+import copy
+
+from src import block_manager
```

Comment:
The `src` folder should be the sources root, so the imports should just be `import block_manager` and `from main import app`